prompting/tasks/__init__.py

```diff
@@ -5,12 +5,15 @@
 from .date_qa import DateQuestionAnsweringTask
 from .generic_instruction import GenericInstructionTask
 from .math import MathTask
+from .mock import MockTask
 
 
 TASKS = {
-    "qa": QuestionAnsweringTask,
-    "summarization": SummarizationTask,
-    "date_qa": DateQuestionAnsweringTask,
-    "debugging": DebuggingTask,
-    "math": MathTask,
+    MockTask.name: MockTask,
```

Comment:
```suggestion
```
We should remove this. It's only for testing anyway, and it messes with our default CLI inputs.

If possible, modify the unit tests to add it manually as an extra task